### PR TITLE
Switch Debugger setup + step 1 of WTR

### PR DESCRIPTION
**Pull request by jasonLaster**
Issue: #1511 

* We were previously loading the debugger bundle as a global. Now it's umd to support loading tests and cleanup the way the debugger code is used.
* We are now maintaining panel.js and index.html inside of the github project because it's nice to track the changes.

This is step one of the web test runner. Step two is refactor helpers.

### CODE REVIEW

**Comment by jasonLaster** [comment]
Yes! There are two simple unit test failures, but the mochitests run fine with this change

File: lerna.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (jasonLaster) [inline lerna.json]: its not needed

File: src/test/integration/tests.js [hunk 2]
```diff
@@ -1,53 +1,10 @@
-const { commands, utils, initDebugger } = require("./head.js");
```
Comment (jasonLaster) [inline src/test/integration/tests.js]: I've got another branch with more mocha stuff which will come later

File: src/test/integration/tests/pretty-print.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    findSource,
+    addBreakpoint,
+    invokeInTab,
+    waitForPaused,
+    stepOver,
+    assertPausedLocation,
+    resume,
+    findElement,
+    ok
+  } = this;
```
Comment (jasonLaster) [inline src/test/integration/tests/pretty-print.js]: these helpers are going to move out of `head.js` in the next PR

File: src/test/mochitest/browser_dbg-pretty-print.js [hunk 4]
```diff
@@ -3,29 +3,5 @@
...
-  yield resume(dbg);
-
-  // The pretty-print button should go away in the pretty-printed
-  // source.
-  ok(!findElement(dbg, "sourceFooter"), "Footer is hidden");
-
-  yield selectSource(dbg, "math.min.js");
-  ok(findElement(dbg, "sourceFooter"), "Footer is hidden");
-});
+const { prettyPrint } = require("devtools/client/debugger/new/integration-tests");
```
Comment (gregtatum) [inline src/test/mochitest/browser_dbg-pretty-print.js]: Are you thinking of every mochitest being a trivial manifest like this to run to require and add the task?

File: webpack.config.js [hunk 5]
```diff
@@ -35,16 +36,18 @@ function buildConfig(envConfig) {
...
   };
 
+  if (!isDevelopment()) {
+    webpackConfig.output.libraryTarget = "umd";
+  }
+
   return toolbox.toolboxConfig(webpackConfig, envConfig);
 }
 
+
```
Comment (gregtatum) [inline webpack.config.js]: nit: whitespacing

File: src/test/integration/tests/pretty-print.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (gregtatum) [inline src/test/integration/tests/pretty-print.js]: nit: whitespacing

File: assets/index.html [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<!DOCTYPE html>
+<html dir="">
+  <head>
```
Comment (gregtatum) [inline assets/index.html]: A small comment intro would be appreciated here about why this file exists, it's not immediately obvious.

File: assets/panel.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
```
Comment (gregtatum) [inline assets/panel.js]: And again here, a small comment would be nice.

File: assets/index.html (hunk 7 above)
Comment (jasonLaster) [inline assets/index.html]: I'll get to this in another cleanup step. I'd like to better document these files holistically

**Pull request merged**